src/game/Globals/CombatCondition.cpp

```diff
@@ -181,6 +181,7 @@ bool CombatConditionMgr::Meets(Unit const* self, int32 Id, float range)
                     case ConditionLogic::AND: if (!results[0] || !results[1]) return false; break;
                     case ConditionLogic::OR:  if (!results[0] && !results[1]) return false; break;
                     case ConditionLogic::XOR: if (results[0] == results[1])  return false; break;
+                    default: break;
```

Comment:
I'm slightly surprised clang complained here. At least in theory it could have seen that the missing NONE was not possible here considering the surrounding code :smile: